lib/brcobranca/remessa/pagamento.rb

```diff
@@ -72,6 +72,10 @@ class Pagamento
       attr_accessor :parcela
       # <b>OPCIONAL</b>: Dias para o protesto
       attr_accessor :dias_protesto
+      # <b>OPCIONAL</b>: de livre utilização pela empresa, cuja informação não é consistida pelo Itaú, e não
+      # sai no aviso de cobrança, retornando ao beneficiário no arquivo retorno em qualquer movimento do título
+      # (baixa, liquidação, confirmação de protesto, etc.) com o mesmo conteúdo da entrada.
+      attr_accessor :uso_da_empresa
```

Comment:
Brcobranca::Remessa::Pagamento#uso_da_empresa is a writable attribute

[Read more about it here.](https://github.com/troessner/reek/blob/master/docs/Attribute.md)